mk/containerized.mk

```diff
@@ -19,7 +19,12 @@ pull: ## Download SVC images
 build:  ## Build SVC images
 build: DOCKER_COMPOSE_ENV = DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1
 
-push: ## Push SVC container images to a registry. Requires previous "docker login"
+GIT_LATEST_TAG := $(shell git tag --sort=taggerdate | tail -1)
+push: ## Push nvmeof and nvmeof-cli containers images to quay.io registries
+	docker tag $(QUAY_NVMEOF):$(VERSION) $(QUAY_NVMEOF):$(GIT_LATEST_TAG)
+	docker push $(QUAY_NVMEOF):$(GIT_LATEST_TAG)
+	docker tag $(QUAY_NVMEOFCLI):$(VERSION) $(QUAY_NVMEOFCLI):$(GIT_LATEST_TAG)
+ 	docker push $(QUAY_NVMEOFCLI):$(GIT_LATEST_TAG)
```

Comment:
If the tag is properly updated in the .env file, we can still use the `docker-compose push` command.

BTW, this doesn't push the `spdk` container. And I think it makes sense not to do so. In this case, we could rewrite the CI not to build it, which would reduce the CI time from 15 minutes to a few minutes. But, same thing, this could be implemented separately.

Reply:
👍🏼